bot/exts/easter/save_the_planet.py

```diff
@@ -1,25 +1,28 @@
 import json
-import random
 from pathlib import Path
 
 from discord import Embed
 from discord.ext import commands
 
+from bot.utils import RandomCycle
+
+EMBED_DATA = []
```

Comment:
This is not what I meant before, you should do this instead.
```suggestion
with Path("bot/resources/easter/save_the_planet.json").open('r', encoding='utf8') as f:
    EMBED_DATA = RandomCycle(json.load(f))
```

Reply:
Oh. I see.